packages/application-shell/src/components/project-suspended/project-suspended.spec.js

```diff
@@ -97,8 +97,10 @@ describe('rendering', () => {
         ],
       }
     );
-    await waitForElement(() =>
-      getByText(/Your Project is temporarily suspended due to maintenance/)
+    await wait(() =>
+      expect(
+        queryByText(/Your Project is temporarily suspended due to maintenance/)
+      ).toBeInTheDocument()
```

Comment:
Hmm isn't this the same thing as before?

Reply:
I found this to be weird too. the other way it was throwing a warning. this way it did not 🤷‍♂